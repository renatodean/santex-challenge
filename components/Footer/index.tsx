import React from "react";
import styled from "@emotion/styled";

const FooterContainer = styled.header`
  background-color: white;
  text-align: center;
  height: 40px;
  padding: 10px 20px;
  margin-top: 10px;
  box-shadow: 0px -1px 2px rgba(0, 0, 0, 0.1);
`;

const Footer = () => (
  <FooterContainer>Santex Challenge - Ecommerce</FooterContainer>
);

export default Footer;
