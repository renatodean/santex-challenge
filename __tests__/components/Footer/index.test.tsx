import { render } from "@testing-library/react";
import Footer from "../../../components/Footer";

describe("Footer snapshot", () => {
  it("renders the Footer", () => {
    const { container } = render(
      <Footer />
    );
    expect(container).toMatchSnapshot();
  });
});
