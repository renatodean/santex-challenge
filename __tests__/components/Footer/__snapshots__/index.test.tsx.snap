// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer snapshot renders the Footer 1`] = `
<div>
  <header
    class="css-1cvg25p"
  >
    Santex Challenge - Ecommerce
  </header>
</div>
`;
